product-entry.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Product Entry</title>
+    <link rel="stylesheet" href="./src/main.css">
+    <link rel="stylesheet" href="./src/product-entry.css">
+    <link href="https://fonts.googleapis.com/css?family=Dancing+Script&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h1>Product Entry</h1>
+        <nav>
+            <a href='./index.html'>Welcome page</a>
+            <a href='./products.html'>Crystal shop</a>
+            <a href="./shopping-cart.html">Shopping cart</a>
+            <a href="./product-entry.html">Product Entry</a>
+        </nav>
+    </header>
+    <form id="product-entry">
+        <p>
+            <label for="code"> Code:
+                <input type="text" name="code" value="code" id="code" required></label>
```

Comment:
If you give an element a starting value attribute it will show up as the starting value when the page loads. I'm not sure that's what you wanted to do here... Check the console when you load this page.